docs/usage/9-getting-deterministic-logical-factories.md

```diff
@@ -0,0 +1,140 @@
+# Getting Deterministic Logical Values
+
+In this following example we create a factory for a `Person` model
+
+```py
+from pydantic import BaseModel
+
+
+class Person(BaseModel):
+    name: str
+    age: int
+```
+
+If we use the factory class:
+
+```py
+from pydantic_factories import ModelFactory
+from pydantic import BaseModel
+
+
+class Person(BaseModel):
+    name: str
+    age: int
+
+
+class PersonFactory(ModelFactory):
+    __model__ = Person
+```
+
+We can get values like this:
+
+```json
+{
+  "values": [
+    {
+      "name": "GMPXMVbyXHUfymCDaloV",
+      "age": 6380
+    },
+    {
+      "name": "iPhGgaTklZTpXCJkWJYU",
+      "age": 4895
+    },
+    {
+      "name": "bvWnrQEEKdspKjyjnWXs",
+      "age": 6623
+    }
+  ]
+}
+```
+
+So we have 2 problems with that.<br>
```

Comment:
Don't use HTML tags in Markdown files (exceptions apply but this context isn't an exception, for line breaks use a simple newline instead.

Rewrite this section like this:
```diff
- So we have 2 problems with that.<br>
- The next run will lead to completely different values.<br>
- The values are not logical (age=6623)
+ Although we do get a proper serialised JSON response as expected, the caveat is the
+ next time the model is serialised, the values will be different! The next section of
+ the docs takes a more comprehensive dive into making the serialisation more
+ deterministic.
```

It would be nice to have a proper explanation on why the serialisation isn't deterministic. As a first-time user of the `pydantic-factories` I would expect the documentations to prepare me for any surprises I might stumble across in the future.